certblob/generate.sh

```diff
@@ -4,16 +4,18 @@ set -euo pipefail
 shopt -s nullglob globstar
 
 # wincrypt.h was downloaded from the following URL:
-# https://raw.githubusercontent.com/reactos/reactos/8ae8083378546fc7d907e489aeebddbe3c8d9399/sdk/include/psdk/wincrypt.h
-# Note that wincrypt.h only has property ID's from Windows XP.
-# certenroll.h appears to have more (from Windows Vista) but it's not in ReactOS.
+# https://github.com/mingw-w64/mingw-w64/blob/2059ac24b9433da7917502ea8fb9e7b9531a2251/mingw-w64-headers/include/wincrypt.h
+# Note that wincrypt.h only has property ID's from Windows Vista.
+# wincrypt.h is implemented from  MinGW-w64 and has property ID's upto 127.
```

Comment:
`upto` looks like a typo.  Also `from  MinGW-w64` has a double space.

Reply:
Fixed the typo and double spacing